lib/xcpretty/formatters/formatter.rb

```diff
@@ -53,6 +53,11 @@ def format_error(message);                                EMPTY; end
     def format_undefined_symbols(message, symbol, reference); EMPTY; end
     def format_duplicate_symbols(message, file_paths);        EMPTY; end
     def format_warning(message);                            message; end
+
+    # TODO: see how we can unify format_error and format_compile_error,
+    #       the same for warnings
+    def format_compile_warning(file_name, file_path, reason,
+                               line, cursor);                 EMPTY; end
```

Comment:
`end` at 60, 69 is not aligned with `def` at 59, 4<br>Unnecessary spacing detected.